test/setup/platforms/kind.ts

```diff
@@ -59,7 +59,7 @@ async function download(osDistro: string): Promise<void> {
   } catch (error) {
     console.log('Downloading KinD...');
 
-    const url = `https://github.com/kubernetes-sigs/kind/releases/download/v0.7.0/kind-${osDistro}-amd64`;
+    const url = `https://github.com/kubernetes-sigs/kind/releases/download/v0.8.1/kind-${osDistro}-amd64`;
```

Comment:
Let's make this commit a chore as it affects only our tests, not worthy of a new feature to customers